src/components/create_cluster/index.js

```diff
@@ -23,27 +23,31 @@ class CreateCluster extends React.Component {
     clusterName: 'My cluster',
     scaling: {
       min: 3,
+      minValid: true,
       max: 3,
+      maxValid: true,
     },
     submitting: false,
     valid: false, // Start off invalid now since we're not sure we have a valid release yet, the release endpoint could be malfunctioning.
     error: false,
   };
 
-  updateAvailabilityZonesPicker = numberPicker => {
+  updateAvailabilityZonesPicker = n => {
     this.setState({
       availabilityZonesPicker: {
-        value: numberPicker.value,
-        valid: numberPicker.valid,
+        value: n.value,
+        valid: n.valid,
       },
     });
   };
 
   updateScalingMin = n => {
     this.setState({
       scaling: {
-        min: n,
+        min: n.value,
+        minValid: n.valid,
```

Comment:
Out of curiosity, why we use here `n.valid` instead of `this.state.scaling.minValid` ?

Reply:
This is the callback that gets called from the minimum scaling number picker when a value changes.

it returns the value, and whether or not that value is valid. So we want to update the state, instead of re-use what was already in the state.

Reply:
ah ok, so actually the number picker is made of two inputs.